content/en/docs/concepts/storage/storage-classes.md

```diff
@@ -242,59 +263,55 @@ allowedTopologies:
 
 ## Parameters
 
-Storage Classes have parameters that describe volumes belonging to the storage
-class. Different parameters may be accepted depending on the `provisioner`. For
-example, the value `io1`, for the parameter `type`, and the parameter
-`iopsPerGB` are specific to EBS. When a parameter is omitted, some default is
-used.
+StorageClasses have parameters that describe volumes belonging to the storage
+class. Different parameters may be accepted depending on the `provisioner`.
+When a parameter is omitted, some default is used.
 
 There can be at most 512 parameters defined for a StorageClass.
 The total length of the parameters object including its keys and values cannot
 exceed 256 KiB.
 
 ### AWS EBS
 
+<!-- maintenance note: OK to remove all mention of awsElasticBlockStore once the v1.27 release of
+Kubernetes has gone out of support -->
+
+Kubernetes {{< skew currentVersion >}} does not include a `awsElasticBlockStore` volume type.
```

Comment:
This is IMO not accurate. Kubernetes indeed does not have AWS EBS volume plugin and we should strongly recommend users to use the CSI driver, however, we still support the old volume plugin API. A StorageClass with `provisioner: kubernetes.io/aws-ebs` still works, however, all the storage work is silently routed to the CSI driver. 

This applies to all cloud-based volume plugins in 1.29.

Reply:
OK. What's the accurate wording to tell users?

If they need to set up an add-on for `awsElasticBlockStore` to appear to work, perhaps we should signpost them to a page about that (that advice is yet to be written), or to https://kubernetes.io/docs/concepts/storage/volumes/#awselasticblockstore?

Reply:
Yes, some wording in https://kubernetes.io/docs/concepts/storage/volumes/#awselasticblockstore would be appropriate. We should somehow say:

* The AWS EBS CSI driver must be installed for PVs with `awsElasticBlockStore` and StorageClasses with `provisioner: kubernetes.io/aws-ebs` to work. Even with the CSI driver.
* Kubernetes keeps this API only for backward compatibility to keep old PVs and StorageClasses working.
* We strongly suggest using the CSI driver directly in any new deployments.

Reply:
Sounds like a separate PR - would you or another contributor from SIG Storage be willing to file an issue to that effect @jsafrane ?